azure-pipelines.yml

```diff
@@ -94,10 +94,11 @@ jobs:
           - script: |
                 if [[ $JHI_JDK = '11' ]]; then
                     echo '*** Using OpenJDK 11'
-                    sudo add-apt-repository ppa:openjdk-r/ppa
-                    sudo apt-get update
-                    sudo apt-get install -y openjdk-11-jdk
-                    sudo update-java-alternatives -s java-1.11.0-openjdk-amd64
+                    curl -s get.sdkman.io | bash
```

Comment:
Plz, can you do the same for:
- the Travis file, although it is currently not use
- the Dockerfile

Reply:
So, I should change this part too, right ?
https://github.com/hipster-labs/jhipster-daily-builds/blob/master/azure-template.yml#L29-L34

Reply:
Updated the travis and docker files as well. 

I guess if you want to check the daily status you should change it otherwise we can wait for SB 2.1.4